app/models/orms/items.py

```diff
@@ -0,0 +1,78 @@
+from sqlalchemy import Column, DateTime, Integer, String
+from typing import List, Optional
+
+from .base import Base
+from ..db import seq
+
+# TODO import Requests and Images
+#from .request import Requests
+#from .images import Images
+
+class Items(Base):
+    __tablename__ = 'items'
+
+    item_id = Column(Integer, seq, primary_key=True)
+    storage_id = Column(Integer, nullable=False)
+    name_ru = Column(String(length=127))
+    name_en = Column(String(length=127))
+    desc_ru = Column(String(length=511))
+    desc_en = Column(String(length=511))
```

Comment:
If you write in swagger models that field is required write in ORM that field isn't nullable.